code/ATMOSPHERICS/components/unary/vent_pump.dm

```diff
@@ -269,6 +274,23 @@
 	if (istype(W, /obj/item/weapon/wrench)&& !(stat & NOPOWER) && on)
 		user << "<span class='danger'>You cannot unwrench this [src], turn it off first.</span>"
 		return 1
+	if (istype(W, /obj/item/weapon/reagent_containers))

```

Comment:
oh yeah hold on let us check each fucking time we are pumping if there are reagents pfhahaha